src/state/apis/zapper/client.ts

```diff
@@ -0,0 +1,984 @@
+    path: '/v2/apps',
+    requestFormat: 'json',
+    response: z.void(),
+  },
+  {
+    method: 'get',
+    path: '/v2/apps/:appSlug',
+    requestFormat: 'json',
+    parameters: [
+      {
+        name: 'appSlug',
+        type: 'Path',
+        schema: z.string(),
+      },
+    ],
+    response: z.void(),
+  },
+  {
+    method: 'get',
+    path: '/v2/apps/:appSlug/balances',
+    requestFormat: 'json',
+    parameters: [
+      {
+        name: 'appSlug',
+        type: 'Path',
+        schema: z.string(),
+      },
+      {
+        name: 'addresses[]',
+        type: 'Query',
+        schema: z.array(z.string()),
+      },
+      {
+        name: 'network',
+        type: 'Query',
+        schema: z
+          .enum([
+            'ethereum',
+            'polygon',
+            'optimism',
+            'gnosis',
+            'binance-smart-chain',
+            'fantom',
+            'avalanche',
+            'arbitrum',
+            'celo',
+            'harmony',
+            'moonriver',
+            'bitcoin',
+            'cronos',
+            'aurora',
+            'evmos',
+          ])
+          .optional(),
+      },
+    ],
+    response: z.void(),
+  },
+  {
+    method: 'get',
+    path: '/v2/apps/:appSlug/positions',
+    description: `Retrieve positions (non-tokenized) for a given application`,
+    requestFormat: 'json',
+    parameters: [
+      {
+        name: 'appSlug',
+        type: 'Path',
+        schema: z.string(),
+      },
+      {
+        name: 'network',
+        type: 'Query',
+        schema: z.enum([
+          'ethereum',
+          'polygon',
+          'optimism',
+          'gnosis',
+          'binance-smart-chain',
+          'fantom',
+          'avalanche',
+          'arbitrum',
+          'celo',
+          'harmony',
+          'moonriver',
+          'bitcoin',
+          'cronos',
+          'aurora',
+          'evmos',
+        ]),
+      },
+      {
+        name: 'groupId',
+        type: 'Query',
+        schema: z.string(),
+      },
+    ],
+    response: z.void(),
+  },
+  {
+    method: 'get',
+    path: '/v2/apps/:appSlug/tokens',
+    description: `Retrieve tokens for a given application`,
+    requestFormat: 'json',
+    parameters: [
+      {
+        name: 'appSlug',
+        type: 'Path',
+        schema: z.string(),
+      },
+      {
+        name: 'network',
+        type: 'Query',
+        schema: z.enum([
+          'ethereum',
+          'polygon',
+          'optimism',
+          'gnosis',
+          'binance-smart-chain',
+          'fantom',
+          'avalanche',
+          'arbitrum',
+          'celo',
+          'harmony',
+          'moonriver',
+          'bitcoin',
+          'cronos',
+          'aurora',
+          'evmos',
+        ]),
+      },
+      {
+        name: 'groupId',
+        type: 'Query',
+        schema: z.string(),
+      },
+    ],
+    response: z.void(),
+  },
+  {
+    method: 'get',
+    path: '/v2/balances/apps',
```

Comment:
The only schema that's actually close to the upstream schema - other endpoints are just auto-generated from the OpenAPI schema, which is missing response schemas 😢. `/v2/balances/apps` should be the only one we initially need however